swap/src/protocol/bob.rs

```diff
@@ -210,7 +212,8 @@ pub enum OutEvent {
     SwapResponse(alice::SwapResponse),
     Message0(Box<alice::Message0>),
     Message1(Box<alice::Message1>),
-    Message2(alice::Message2),
+    Message2,
```

Comment:
What is the "Event" of `Message2`? Given that it doesn't have any content, I assume it is `Message2Sent`? These variants could use better names :)

Reply:
These will be scraped when I use `libp2p-async-await` :)

Reply:
Fair enough, still some feedback to consider for the future: Naming any event should likely include a verb in simple past tense.